pkg/cidata/cidata.TEMPLATE.d/boot/05-lima-disks.sh

```diff
@@ -28,6 +28,13 @@ for i in $(seq 0 $((LIMA_CIDATA_DISKS - 1))); do
 		fi
 	fi
 
+  if command -v growpart >/dev/null 2>&1 && command -v resize2fs >/dev/null 2>&1; then
+    growpart "/dev/${DEVICE_NAME}" 1 || true
+    # Only resize when filesystem is in a healthy state
+    if e2fsck -f -p "/dev/disk/by-label/lima-${DISK_NAME}"; then
+      resize2fs "/dev/disk/by-label/lima-${DISK_NAME}" || true
```

Comment:
Should probably check for an ext2 (et al) filesystem, perhaps also support xfs ?

Reply:
Apparently `-d` grows to the biggest.

Reply:
`xfs_repair` repairs FS but `e2fsck` just only checks it. I think need to use `fsck.$FSTYPE`

Reply:
I added `xfs_growfs` and tested `ext2/3/4` and `xfs`

Reply:
I thought you wanted to do a repair, since you did `e2fsck -p`

Reply:
I did not plan to repair the file system, but only to check the functionality before increasing the size. Same code can be found at [04-persistent-data-volume.sh](https://github.com/lima-vm/lima/blob/bfd74711f565fe4cbd9cf923b96e6a11e4c785ca/pkg/cidata/cidata.TEMPLATE.d/boot/04-persistent-data-volume.sh#L24)